ViewPumpSample/build.gradle

```diff
@@ -23,7 +23,7 @@ android {
     }
 
     lintOptions {
-        disable 'AppCompatCustomView'
+        abortOnError false
```

Comment:
This is just the sample app, we don't care about lint warnings here. Most of them were things like GradleDependency or CustomView